frontend/src/views/assignments/offer-actions.js

```diff
@@ -59,6 +59,15 @@ function OfferActionButtons(props) {
     } = props;
 
     function createOffers() {
+        if (
+            selectedAssignments.some(
+                (assignment) => assignment.active_offer_status === "withdrawn"
+            )
+        ) {
+            window.confirm(
```

Comment:
Do we want to use native window confirm dialog here or should we use react bootstrap's Modal with extra information displayed?

Reply:
Don't worry. I will change to React bootstrap after you have finished your PR since we are modifying the same file.